pkg/service/presentation/service.go

```diff
@@ -27,7 +30,7 @@ import (
 )
 
 type Service struct {
-	storage    *Storage
+	storage    presentationstorage.Storage
```

Comment:
should this still be a ptr?

Reply:
No. `presentationstorage.Storage` is an interface.